src/everest/simulator/everest_to_ert.py

```diff
@@ -112,13 +113,15 @@ def _extract_summary_keys(
 
     try:
         data_wells = list(_load_all_wells(data_files))
-    except Exception:
+    except Exception as e:
         warn_msg = (
             "Failed to load well names from {}. "
             "Only well data for wells specified in config file will be "
-            "internalized during run."
+            "internalized during run. Full error: {}"
         )
-        logging.getLogger(EVEREST).warning(warn_msg.format(data_files))
+        full_message = warn_msg.format(data_files, traceback.format_exception(e))
```

Comment:
Will the user get a full traceback now? That is probably a bit excessive, can we trim it down?

Reply:
Logging the full exception is great when debugging or running --debug and it should not be trimmed. There is a difference what is beeing logged and whats beeing displayed to user

Reply:
At least previously this was also shown to the user, in which case we should not show the full traceback, but fine to log the full traceback.